src/contract_wrappers/ether_token_wrapper.ts

```diff
@@ -13,9 +13,11 @@ import {artifacts} from '../artifacts';
 export class EtherTokenWrapper extends ContractWrapper {
     private _etherTokenContractIfExists?: EtherTokenContract;
     private _tokenWrapper: TokenWrapper;
-    constructor(web3Wrapper: Web3Wrapper, tokenWrapper: TokenWrapper) {
+    private _contractAddress?: string;
```

Comment:
`IfExists` --> Do this for all of the new instance variables and constructor args.